galaxy.yml

```diff
@@ -16,49 +14,50 @@ version: 1.0.0
 readme: README.md
 
 # A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
 authors:
-  - IBM Lab for SAP Solutions
-  - IBM Cloud for SAP
-  - IBM Consulting for SAP
+- Ondra Machacek <omachace@redhat.com>
 
 ### OPTIONAL but strongly recommended
 # A short summary description of the collection
-description: Collection of Ansible Modules and Ansible Roles for SAP system operations
+description: Ansible collection to handle day 2 operational SAP administration tasks.
 
 # Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
 # accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
 license:
-- Apache-2.0
+- Apache 2.0
 
 # The path to the license file for the collection. This path is relative to the root of the collection. This key is
 # mutually exclusive with 'license'
 license_file: ''
 
 # A list of tags you want to associate with the collection for indexing/searching. A tag name has the same character
 # requirements as 'namespace' and 'name'
-tags: []
+tags:
+  - linux
+  - sap
 
 # Collections that this collection requires to be installed for it to be usable. The key of the dict is the
 # collection label 'namespace.name'. The value is a version range
 # L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
 # range specifiers can be set and are separated by ','
-dependencies:
-  community.sap_libs: '*'
+dependencies: {}
 
 # The URL of the originating SCM repository
-repository: https://github.com/sap-linuxlab/community.sap_operations
+repository: https://github.com/sap-linuxlab/sap_operations
 
 # The URL to any online docs
-documentation: https://github.com/sap-linuxlab/sap-linuxlab.github.io/blob/master/README.md
+documentation: https://github.com/sap-linuxlab/sap_operations
 
 # The URL to the homepage of the collection/project
-homepage: https://sap-linuxlab.github.io
+homepage: https://github.com/sap-linuxlab/sap_operations
 
 # The URL to the collection issue tracker
-issues: https://github.com/sap-linuxlab/community.sap_operations/issues
+issues: https://github.com/sap-linuxlab/sap_operations/issues
```

Comment:
wrong repo

Reply:
There has been a lot of discussions about repos, collections names, etc.

This is unfortunate artifact of all these last minute changes. TBH it is not quite clear for me where this collection will reside at the end of the day.